schemas/manifest.cue

```diff
@@ -25,27 +25,41 @@
 
 // containerd includes constraints from #dep and tighter bounds on version
 #containerd: #dep & {
-	versions: [...#containerd_ver]
+    versions: [...#containerd_ver]
     edge: #containerd_ver
 }
 
 #runc_ver: =~"[0-9]+.[0-9]+.[0-9]+-(rc)?[0-9]+" // rc92,rc95 previously used.
 
 #runc: #dep & {
-	versions: [...#runc_ver]
+    versions: [...#runc_ver]
 }
 
 #kubernetes_ver: =~"[0-9]+.[0-9]+.[0-9]+(-hotfix.[0-9]{8})"
 
 #kubernetes: #dep & {
-	versions: [...#kubernetes_ver]
+    versions: [...#kubernetes_ver]
+}
+
+#kube_proxy_dep: {
+    downloadURL: string
+    amd64OnlyVersions: [...string]
+    multiArchVersions: [...string]
+}
+
+#kube_proxy: {
+    dockerKubeProxyImages: string | null
+    containerdKubeProxyImages: {
+        ContainerImages: [...#kube_proxy_dep]
+    }
 }
 
 // root object schema enforced against manifest.json
 #root: {
-	runc:                       #runc
-	containerd:                 #containerd
-	[string]:                   #dep
+    runc:                                       #runc
+    containerd:                                 #containerd
+    "kube-proxy":                               #kube_proxy
```

Comment:
why is this one a string ?

Reply:
cue export command wasn't parsing the fiile correctly when dash is included in the key. I was using dash for consistency, but if you prefer I can change it to camelcase to remove the double quote. Let me know if you think that's preferred. 

Same issue with using jq with dashed key (below comments).

Reply:
I would opt to keep it consistent and remove the `-` to avoid quoting